src/app/containers/ContactsContainer.js

```diff
@@ -250,7 +254,7 @@ const ContactsContainer = ({ location, history }) => {
     );
 
     return (
-        <PrivateLayout>
+        <PrivateLayout title={contactGroup ? `${contactGroup.Name} - ProtonContacts` : `Contacts - ProtonContacts`}>
```

Comment:
Should `ProtonContacts` be included in the translation?

An alternative is to always append it in the `useEffect`?